site/src/testHelpers/localstorage.ts

```diff
@@ -0,0 +1,22 @@
+export const localStorageMock = () => {
+  const store = {} as Record<string, string>
+
+  return {
+    getItem: (key: string): string => {
+      return store[key]
+    },
+    setItem: (key: string, value: string) => {
+      store[key] = value
+    },
+    clear: () => {
+      Object.keys(store).forEach((key) => {
+        delete store[key]
+      })
+    },
+    removeItem: (key: string) => {
+      delete store[key]
+    },
+  }
+}
+
+Object.defineProperty(window, "localStorage", { value: localStorageMock() })
```

Comment:
Since this is being executed when the module is imported, maybe can be interesting to move all this code into the test setup file and mock it globally. We are doing that for a few default window interfaces.

Reply:
maybe, I just wanted to avoid putting things in the global if my test was the only one using it atm.